queries/gdscript/highlights.scm

```diff
@@ -123,4 +155,186 @@
   "puppetsync"
 ] @keyword
 
-"return" @keyword.return
+"func" @keyword.function
+
+[
+  "await"
```

Comment:
Just a note for #4382, will move this to `@keyword.coroutine` once this is merged

Reply:
I have subscribed to #4382.